recipes/carma/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -xeo pipefail
+
+mkdir build
+cd build
+
+cmake ${CMAKE_ARGS} \
+    -DCMAKE_INSTALL_INCLUDEDIR=${PREFIX}/include \
+    -DCMAKE_INSTALL_DATAROOTDIR=${PREFIX}/share \
+    -DCARMA_INSTALL_LIB:BOOL=ON \
+    ..
+
+cmake --build . --config Release --target install
```

Comment:
I'm assuming there is no `ctest` or `make check`, right?

Reply:
Ha, good point! I'll double check in the feedstock.